### PR TITLE
Add zellij support

### PR DESCRIPTION
**Pull request by albheim**
Zellij is a terminal multiplexer like tmux or screen, and I (as well as others #329) think it would be great if vim-slime could support it. 

I had a quick try at implementing the needed functionality for vim-slime to work by using the reasonably new `action` subcommand in zellij.
I have never really tried vim-script before other than very basic vimrc stuff, so the current implementation is mostly copy pasted from the other implementations in vim-slime glued together with some guessing.
It seems to mostly work with regards to the zellij interaction, though I had some weird characters popping up in the vim window when running under WSL that I don't get for tmux, but on my regular linux install it all works fine.

You can set the session name you want to send to, default is `current`, as well as the relative pane in relation to the currently active pane, default is `current`, other options are `up/down/left/right` since those are the keywords available for pane navigation.

I have not yet added anything to the readme or documentation, thought I would first check if this would be acceptable? And if more should be done, get an idea of exactly what is required.

### CODE REVIEW

**Comment by jpalardy** [comment]
Hi @albheim 

Thanks for the PR ✅ 

It's looking good, a few questions:
- if it needs support, will you be willing to pitch in? (I would cc on issues)
- can you add some documentation in both `README.md` and `doc/vim-slime.txt`? (it doesn't have to be long, check what other targets have done)

**Comment by albheim** [comment]
I wouldn't mind having a look if you cc.

Added some docs now, do you have a preference where in the order they are added? Currently I just squeezed them in after kitty since that is where I put the implementation.

File: autoload/slime.vim [hunk 1]
```diff
@@ -25,7 +25,8 @@ function! s:ScreenSend(config, text)
         \ " -X eval \"readreg p " . g:slime_paste_file . "\"")
   call system("screen -S " . shellescape(a:config["sessionname"]) . " -p " . shellescape(a:config["windowname"]) .
         \ " -X paste p")
-  call system('screen -X colon ""')
+  call system('screen -X colon "
+"')
```
Comment (jpalardy) [inline autoload/slime.vim:29]: is that a merge/reformat include?

let's revert it?

**Review by jpalardy — COMMENTED** [review]
looking good 🚀 

I left a small comment to revert the invisible newline 😄

**Comment by jpalardy** [comment]
> Currently I just squeezed them in after kitty since that is where I put the implementation.

that's fine — I'm especially grateful for the documentation patch, I won't be picky about the where 👍

File: autoload/slime.vim (hunk 1 above)
Comment (albheim) [reply autoload/slime.vim:29]: Ahh, yeah. Had the same problem for the initial commit, but then I noticed it. Forgot about it for this small fix.

**Comment by jpalardy** [comment]
Thanks again 😄

**Pull request merged**